src/styles/card.scss

```diff
@@ -0,0 +1,56 @@
+.card {
+  display: flex;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+
+  &-image {
+    box-sizing: border-box;
+  }
```

Comment:
will be applied anyway (you set it in the `*` selector), so you should remove it here